coda/coda_mdstore/factories.py

```diff
@@ -0,0 +1,80 @@
+"""
+Coda MDStore Model factories for test fixtures.
+"""
+import factory
+
+from datetime import datetime
+
+from . import models
+
+
+class BagFactory(factory.django.DjangoModelFactory):
+
+    class Meta:
+        model = models.Bag
+
+    name = factory.Sequence(lambda n: 'ark:/00001/id{0}'.format(n))
+    files = 10
+    size = 139023
+    bagit_version = '3.4'
+    last_verified_date = datetime.now()
+    last_verified_status = 'fake status'
+    bagging_date = datetime.now()
+
+
+class Bag_InfoFactory(factory.django.DjangoModelFactory):
+
+    class Meta:
+        model = models.Bag_Info
+
+
+class ExternalIdentifierFactory(factory.django.DjangoModelFactory):
+
+    class Meta:
+        model = models.External_Identifier
+
+    value = factory.Sequence(lambda n: 'value {0}'.format(n))
+
+
+class FullBagFactory(BagFactory):
+    """
+    Factory to create a Bag object with two related Bag_Info objects, and
+    2 related External_Identifier objects.
+    """
+    info1 = factory.RelatedFactory(
```

Comment:
i disagree :scream_cat: 

I usually only add the blank line below the docstring if a method follows. For class variables, I like them to be directly under the docstring. PEP8 is vague about this.